src/WorldIDIdentityManagerImplV1.sol

```diff
@@ -373,7 +374,8 @@ contract WorldIDIdentityManagerImplV1 is WorldIDImpl, IWorldID {
         uint256 preRoot,
         uint32 startIndex,
         uint256[] calldata identityCommitments,
-        uint256 postRoot
+        uint256 postRoot,
+        uint32 opGasLimit
```

Comment:
This also requires changes to signup-sequencer.

Could we instead make it a field of the state bridge contract and allow changing it by owner? Our L2 calls should in theory have constant gas cost, no?